src/dymaptic.GeoBlazor.Core/Components/Renderers/RasterStretchRenderer.cs

```diff
@@ -155,6 +155,10 @@ internal override void ValidateRequiredChildren()
         base.ValidateRequiredChildren();
     }
 
+    void IImageryRenderer.ValidateRequiredChildren()
```

Comment:
replace the interface with a base class (see notes there) and then you shouldn't need this duplicate `ValidateRequiredChildren`

Reply:
removed and successfully tested